astroquery/solarsystem/pds/tests/test_pds_remote.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+import numpy as np
+import astropy.units as u
+
+from ... import pds
+
+
+@pytest.mark.remote_data
+class TestRingNodeClass:
+    def test_ephemeris_query(self):
+
+        bodytable, ringtable = pds.RingNode.ephemeris(
+            planet="Uranus",
+            epoch="2022-05-03 00:00",
+            location=(-120.355 * u.deg, 10.0 * u.deg, 1000 * u.m),
+        )
+        # check system table
+        systemtable = bodytable.meta
+        assert np.allclose(
+            [
+                -56.12233,
+                -56.13586,
+                -56.13586,
+                -56.01577,
+                0.10924,
+                354.11072,
+                354.12204,
+                2947896667.0,
+                3098568884.0,
+                10335.713263,
+            ],
+            [
+                systemtable["sub_sun_lat"].to(u.deg).value,
+                systemtable["sub_sun_lat_min"].to(u.deg).value,
+                systemtable["sub_sun_lat_max"].to(u.deg).value,
+                systemtable["opening_angle"].to(u.deg).value,
+                systemtable["phase_angle"].to(u.deg).value,
+                systemtable["sub_sun_lon"].to(u.deg).value,
+                systemtable["sub_obs_lon"].to(u.deg).value,
+                systemtable["d_sun"].to(u.km).value,
+                systemtable["d_obs"].to(u.km).value,
+                systemtable["light_time"].to(u.second).value,
+            ],
+            rtol=1e-2,
+        )
```

Comment:
Here and below, too:

please refrain from using black for astroquery, and condense these 25 lines into something sensible, all of these should fit into ~5 lines.